packages/preview-middleware/src/ui5/middleware.ts

```diff
@@ -47,11 +47,16 @@ async function createRouter(
     if (config.adp) {
         await initAdp(resources.rootProject, config.adp, flp, middlewareUtil, logger);
     } else {
-        const manifest = await resources.rootProject.byPath('/manifest.json');
-        if (manifest) {
-            await flp.init(JSON.parse(await manifest.getString()));
+        const flpPath = config.flp.path ?? '/test/flpSandbox.html';
+        if (await resources.rootProject.byPath(flpPath)) {
+            logger.info('File to render preview found on file system.');
         } else {
-            throw new Error('No manifest.json found.');
+            const manifest = await resources.rootProject.byPath('/manifest.json');
```

Comment:
I just noticed, this is wrong, we need to adapt the function https://github.com/SAP/open-ux-tools/blob/f79c7a1ead5577b50fe35f8fd64ee64d9c1e60cc/packages/preview-middleware/src/base/flp.ts#L346

Reply:
Yeah, that change needs to be reverted because it won't work for adaptation projects. The manifest needs to be provided during init.

Reply:
Actually, I am confused what this change does..

Reply:
Sorry, 3rd comment (deleted the other 2), but it should be later in the routes, when the file is loaded. There should already be code that handles the situation and returns the local file if present.

Reply:
thanks @tobiasqueck will do.

Reply:
https://github.com/SAP/open-ux-tools/pull/1930/commits/9cd77089776dfb45a2138bc98d3866a4ef5c6fd1